scripts/install.py

```diff
@@ -3567,7 +3659,72 @@ def main():
         default='',
         help=f'Delete the oldest indices when the database exceeds this threshold (e.g., 250GB, 1TB, 60٪, etc.)',
     )
-
+    storageArgGroup.add_argument(
+        '--ilm-enable',
+        dest='ilmPolicy',
+        type=str2bool,
+        metavar="true|false",
+        nargs='?',
+        const=True,
+        default=False,
+        help="Should Arkime use an ILM policy? (see https://https://arkime.com/faq#ilm)",
+    )
+    storageArgGroup.add_argument(
+        '--ilm-hot-warm-enable',
+        dest='ilmHotWarm',
+        type=str2bool,
+        metavar="true|false",
+        nargs='?',
+        const=True,
+        default=False,
+        help="'Should Arkime use a hot/warm design in which non-session data is stored in a warm index? (see https://https://arkime.com/faq#ilm)",
```

Comment:
since you put the URL in the previous argument (which is great) I think you can leave it off here